<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceAnalyzerGUI</class>
 <widget class="QMainWindow" name="SourceAnalyzerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Left Spectrum</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Spectrum</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Correlation</string>
    </property>
   </widget>
   <widget class="QListView" name="source_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select source</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="file_box">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>370</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Set output device</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filename_text">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>160</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="start_file_button">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QListView" name="output_device_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>140</width>
       <height>60</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="correlation_plot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>351</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="left_plot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="right_plot" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>118</width>
      <height>27</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>mm:ss.zzz</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="device_box">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>370</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Set input device</string>
    </property>
    <widget class="QListView" name="input_device_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="start_device_button">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="circle_plot" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="angleText">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="dynamic_plot" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="ildText">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="itdText">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">./src/plot/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
